packages/zwave-js/src/lib/log/Driver.ts

```diff
@@ -53,6 +54,7 @@ export class DriverLogger extends ZWaveLoggerBase {
 			level: actualLevel,
 			message,
 			direction: getDirectionPrefix("none"),
+			context: CONTEXT,
```

Comment:
```suggestion
			context: { ...CONTEXT },
```
would avoid a situation where applications can change our `CONTEXT` object's contents.